src/ontology/edits/oeo-physical.omn

```diff
@@ -2871,6 +2872,54 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130",
         <http://openenergy-platform.org/ontology/oeo/OEO_00000316>
     
     
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00230000>
+
+    Annotations: 
+        <http://openenergy-platform.org/ontology/oeo/definition> "Storage capacity is the maximum energy an energy storage object can store."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/320"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "pull request: https://github.com/OpenEnergyPlatform/ontology/pull/391",
+        rdfs:label "storage capacity"@en
```

Comment:
can you delete this "@en" label? I don't know how do it, but Anna knows.

Reply:
I did it.